source/code/plugin/kubernetes_container_inventory.rb

```diff
@@ -273,12 +273,12 @@ def obtainWindowsContainerEnvironmentVars(pod, container)
                   value = "secretKeyRef_#{secretName}_#{secretKey}"
                 end
               else
-                value = envVar["valueFrom"].to_s
+                value = envVar["valueFrom"].join('')
```

Comment:
similar all other places .. we shouldn't use join('') and keep the JSON representation to break JSON based tools.  It might work in the UX since removed the usage of JSON parser with the custom parser.